lua/graph.lua

```diff
@@ -1305,6 +1314,40 @@ function filter_tags_generic(kv)
     kv["bus_backward"] = "true"
   end
 
+  --let all the :forward overrides through
+  local mv_forward = kv["motor_vehicle:forward"] or kv["vehicle:forward"]
```

Comment:
hm, being strict `motor_vehicle` does not equal `vehicle`, I think the most important distinction for us is that `vehicle` includes bicycle (but don't ask about e-bikes haha). OSM definition somewhat disagrees with Wikipedia definition (if rail-based is vehicle or not), but I think we should include bike for `vehicle` at least.